builtin/providers/aws/resource_aws_autoscaling_group.go

```diff
@@ -446,7 +446,6 @@ func resourceAwsAutoscalingGroupRead(d *schema.ResourceData, meta interface{}) e
 	d.Set("health_check_type", g.HealthCheckType)
 	d.Set("launch_configuration", g.LaunchConfigurationName)
 	d.Set("load_balancers", flattenStringList(g.LoadBalancerNames))
-	d.Set("target_group_arns", flattenStringList(g.TargetGroupARNs))
```

Comment:
Can you explain the removal here, please? I think it would break things in the sense that Terraform wouldn't be able to detect drifts for this field from the state anymore.